hdr/sqlite_modern_cpp.h

```diff
@@ -399,41 +399,44 @@ namespace sqlite {
 		);
 	}
 	
+	enum class Encoding {
+	  AUTO = SQLITE_ANY,
+	  UTF8 = SQLITE_UTF8,
+	  UTF16 = SQLITE_UTF16
+	};
 	struct sqlite_config {
+    int flags = SQLITE_OPEN_READWRITE | SQLITE_OPEN_CREATE;
```

Comment:
For a `enum class` `operator |` isn't defined, so this leads to quite some boilerplate. Do need `operator&` too?